specfile/tags.py

```diff
@@ -0,0 +1,439 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+import collections
+import itertools
+import re
+from typing import List, Optional, Tuple
+
+from specfile.sections import Section
+
+# valid tag names extracted from lib/rpmtag.h in RPM source
+TAG_NAMES = [
```

Comment:
This should probably be a set (for more efficient check of presence) since we are only checking for presence or iterating over it where order is not relevant.

Reply:
Good point.